src/main/java/mate/academy/Main.java

```diff
@@ -2,32 +2,52 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.util.Optional;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.UserService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
 
-        Movie fastAndFurious = new Movie("Fast and Furious");
-        fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
+    private static final String MAIN_PACKAGE_NAME = "mate.academy";
+    private static final Injector injector =
+            Injector.getInstance(MAIN_PACKAGE_NAME);
+    private static final String MOVIE_TITLE = "Fast and Furious";
+    private static final String MOVIE_DESCRIPTION = "An action film about "
+            + "street racing, heists, and spies.";
+    private static final String FIRST_HALL_CAPACITY = "first hall with capacity 100";
+    private static final String SECOND_HALL_CAPACITY = "second hall with capacity 200";
+    private static final long LONG_1 = 1L;
+    private static final String USER_PASSWORD = "nazar1177";
+    private static final String USER_EMAIL = "nazar@gmail.com";
+    private static final String EMAIL_NOT_EXIST = "nazarss@gmail.com";
```

Comment:
The variable `EMAIL_NOT_EXIST` is declared but never used in the code. If it's not needed, consider removing it to clean up the code.